### PR TITLE
Purge leftover node-sass peer dependency

### PR DESCRIPTION
**Pull request by st3iny**
This will make `npm install` run **A LOT** faster.

Despite switching to sass we still had an instance of node-sass in our package-lock.json.  It is an optional peer dependency of sass-loader but sass-loader works with both the new sass and old node-sass package.

Ref https://github.com/webpack-contrib/sass-loader#getting-started
Ref https://github.com/webpack-contrib/sass-loader/blob/b8d22ad4b85b29a53216860bf38e6f0022979dfb/package.json#L47

TL;DR: This is a safe operation because we are already using the new sass package. As long as `npm install && npm run build` works we're good.

### CODE REVIEW

**Review by tcitworld — APPROVED** [review]
🧹
